build-files/krona/2.8.1/Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM ubuntu:jammy AS app
+
+ARG KRONA_VER="2.8.1"
+ARG KRAKENTOOLS_COMMIT="d4a2fbe2833c2392a724697c0c97a3014e748474"
+
+USER root
+
+WORKDIR /
+
+LABEL base.image="ubuntu:jammy"
+LABEL dockerfile.version="1"
+LABEL software="krona"
+LABEL software.version="${KRONA_VER}"
+LABEL description="Krona Tools is a set of scripts to create Krona charts from several Bioinformatics tools as well as from text and XML files."
+LABEL website="https://github.com/marbl/Krona"
+LABEL license="https://github.com/marbl/Krona/blob/master/KronaTools/LICENSE.txt"
+LABEL maintainer="Sage Wright"
+LABEL maintainer.email="sagemwright@gmail.com"
+
+# 'LABEL' instructions tag the image with metadata that might be important to the user
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    wget \
+    ca-certificates \
+    curl \
+    perl \
+    unzip \
+    python3 \
+    python-is-python3 \
+    python3-pip \
+    python3-setuptools \
+    make \
+    procps && \
+    apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+ENV PATH="/root/miniconda3/bin:$PATH"
+
+RUN wget https://github.com/marbl/Krona/releases/download/v${KRONA_VER}/KronaTools-${KRONA_VER}.tar && \
+    tar -xvf KronaTools-${KRONA_VER}.tar && \
+    rm KronaTools-${KRONA_VER}.tar && \
+    mv KronaTools-${KRONA_VER} KronaTools && \
+    cd KronaTools && \
+    ./install.pl
+
+RUN pip install biopython==1.84 \
+    pandas==2.2.3
+
+ARG KRAKENTOOLS_COMMIT="d4a2fbe2833c2392a724697c0c97a3014e748474"
```

Comment:
Is this a duplicate line or does the ARG need to be redefined at this point?

Reply:
nah this is a duplicate. i'll remove it. thanks!